### DISCUSSION TITLE
Laravel and JWT

### DISCUSSION THREAD

**Discussion by josuelrocha** [laravel/framework#39417] [General]
Hello,

I was thinking about Laravel and JWT and a question came up, why doesn't Laravel adopt JWT as standard authentication for API? Is JWT not considered the standard for authentication APIs on the internet? Is there any specific reason why Laravel developers prefer OAT over JWT?

I appreciate any comments from the community on the subject. :)

**Comment by PaolaRuby** [comment]
https://laravel.com/docs/8.x/sanctum

**Reply by josuelrocha** [reply]
the sanctum package does not use JWT but OAT.

**Reply by PaolaRuby** [reply]
Sorry I've never heard about OAT, just OAUTH and JWT

**Reply by josuelrocha** [reply]
OAT (opaque access token) is a simple token, which needs to be stored in the backend for validation, while JWT does not need to be stored and all the information is inside the token itself and is signed by the backend so that its modification is not possible.

**Reply by PaolaRuby** [reply]
Thanks for the info

**Comment by ghost** [comment]


**Reply by wizzymore** [reply]
Why do you need JWT? Laravel Sanctum works in the same way, only that the token looks different and can't hold data.
